deploy/eck-operator/templates/configmap.yaml

```diff
@@ -79,3 +79,9 @@ data:
     {{- with .Values.webhook.certsSecret }}
     webhook-secret: {{ . }}
     {{- end }}
+    {{- $passwordLength := int (dig "policies" "passwords" "length" 24 .Values.config) }}
```

Comment:
Do we need to redeclare `24` and the default allowed characters? IIUC we should set `password-length` and `password-allowed-characters` only if they are set by the user?